### DISCUSSION TITLE
Improve PDF

### DISCUSSION THREAD

**Discussion by ghwood** [ietf-tools/datatracker#3834] [Ideas]
Currently, when there are no I-Ds associated with a meeting session, clicking the link for materials leads to a blank PDF. E.g.

On IETF 113 meeting agenda (https://datatracker.ietf.org/meeting/113/agenda), clicking on the meeting materials icon leads to:

https://datatracker.ietf.org/meeting/113/agenda/ietf-drafts.pdf

In these cases, it would be friendlier to provide a PDF that explicitly states: 

"There are no drafts associated with this session.”

**Comment by rjsparks** [comment]
It might be better to not offer a link in the first place.

We should also revisit whether a pdf of all the drafts for the session is the right collection of things to be building - it's not clear that this particular utility is getting used.
